packages/govern-console/src/containers/DAO/DaoMainPage.tsx

```diff
@@ -1,3 +1,4 @@
+// TODO: This component is deprecated. Remove.
```

Comment:
Please do. :D

If someone misses it they can always go back in git history. Also, if someone's indeed going to miss it (because something breaks without it), we'll be aware of it sooner!